ui/src/app/views/workspace-list.spec.tsx

```diff
@@ -28,47 +32,67 @@ describe('WorkspaceList', () => {
   const updateCache = jest.fn();
 
   const component = () => {
-    return mount(<WorkspaceList/>);
+    return mount(<WorkspaceList/>, {attachTo: document.getElementById('root')});
```

Comment:
This is what seemed to fix the popup trigger issue for me. I'm still not... quite sure why it works, but it did. I commented on RW-2625, this needs more investigation.